hipe4ml/tree_handler.py

```diff
@@ -269,10 +255,14 @@ def write_df_to_parquet_files(self, base_file_name="TreeDataFrame", path="./", s
             If True and the slices are available, single parquet files for each
             bins are created
         """
-        self._full_data_frame.to_parquet(
-            f"{path}{base_file_name}.parquet.gzip", compression="gzip")
+        if self._full_data_frame is not None:
+            self._full_data_frame.to_parquet(
+                f"{path}{base_file_name}.parquet.gzip", compression="gzip")
```

Comment:
I think we can add a warning when there is no dataframe to save, like what is done for the slices.